domains/efe-esxi.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Efebey2903",
+           "email": "",
+           "discord": "1036244397026705498",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.p80sFWEHwAx-piEPnJxcNPKFf2ybMxNbvJEw_6IYUxLq4jgGjBOO9et5GcBy8xqjV8ltsyiQVBiD4-rI2chW4CekjfrmbtztSEuDFgO_-YonHXnY97NdGXa3vwkBmBcokVlvYDn3hOMROBexby_5imYQ1kbWG17q4430-Adj5l2g8aX1TtEJaDQMZE7cDx48WjRzmpDxWn5bQGrFDLzUXyISzSw6t4Yn3ajD0s4EHoRMW8kTn4Iz2LnHT9c92SCwRzsFzRd7WpeLQ7D0d5allFFAbUWLzar96QHzpex2pMJM3UbZI11RaCTmQJP6js8xGW5huq1O9KzsS4BzUlKesA.cIb7UseFOBaIpZ84z8b_Ng.xTLtMCbl_XRJvrIRcTCD6_z4LfG5x4Udf8Nsrtp51Q8eRdvkEaSaBzdc2TDAAZkGz0H3VUiodCL2fDCF-UI2a7T0rb6cq7T7PEUt66BJ57c.xmrV6jXfP79yzpM-6hCDDQ"
+        },
+    
+        "record": {
+            "A": ["115.239.230.106"]
```

Comment:
Unable to load site, please send a preview.

Reply:
it is ESXi but i didnt boot it so